lib/vanity/adapters/abstract_adapter.rb

```diff
@@ -48,72 +48,81 @@ def destroy_metric(metric)
 
       # -- Experiments --
 
-      def experiment_persisted?(experiment)
+      def experiment_persisted?(experiment_id)
```

Comment:
Since this would be a breaking, non-backward compatible change, can we pull this change into a separate PR? When these changes are made, we also should be included backward compatible code, for example, see https://github.com/assaf/vanity/pull/317. What do you think about keeping these methods accepting `experiment` for this PR?